io.github.sigmasd.nosleep.yml

```diff
@@ -0,0 +1,48 @@
+app-id: io.github.sigmasd.nosleep
+
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Platform
+
+command: nosleep
+
+finish-args: 
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  # Required to download deno url imports (in this case, deno-gtk-py and its dependencies)
+  - --share=network
+
+modules:
+  - name: nosleep
+    buildsystem: simple
+    build-commands:
+      - mkdir /app/bin
+      - cp deno /app/bin
+      - cp main.ts /app/bin/main.ts
+      - cp inhibit.ts /app/bin/inhibit.ts
+      - cp main.css /app/bin/main.css
+      - echo "DENO_PYTHON_PATH=/usr/lib/$(uname -m)-linux-gnu/libpython3.11.so.1.0 CSS=/app/bin/main.css /app/bin/deno run -A --unstable /app/bin/main.ts"  >/app/bin/nosleep
+      - chmod +x /app/bin/nosleep
+
+      - install -Dm644 distro/io.github.sigmasd.nosleep.desktop -t /app/share/applications/
+      - install -Dm644 distro/io.github.sigmasd.nosleep.svg -t /app/share/icons/hicolor/scalable/apps/
+      - install -Dm644 distro/io.github.sigmasd.nosleep.metainfo.xml -t /app/share/metainfo/
+
+    sources:
+      - type: archive
+        url: https://github.com/denoland/deno/releases/download/v1.38.3/deno-x86_64-unknown-linux-gnu.zip
+        sha256: 01eaef0e7a922324cdeb2ce19f9e2b457feb489b568e9da86c7b4cc8a3c4e0a3
+        only-arches:
+          - x86_64
+      # Deno doesn't have official aarch64 releases for now
+      # This third party is well known though
+      - type: archive
+        url: https://github.com/LukeChannings/deno-arm64/releases/download/v1.38.3/deno-linux-arm64.zip
```

Comment:
so you take some random binary...

Reply:
My counter argument to that is that the author has been doing this for a long time by now, and he's actively keeping up with the deno releases, and as a random metric, you can see its mentioned many times in the deno repo https://github.com/search?q=repo%3Adenoland%2Fdeno+%22https%3A%2F%2Fgithub.com%2FLukeChannings%2Fdeno-arm64%2F%22&type=issues 
The second arguments is: the app run in the flatpak sandbox so does it really matter

For other options I thought of:
- dropping aarch64 , is that acceptable ?
- compiling deno (and maybe make it a baseapp) but that's going to need some  good amount of work (and patience for the ci)